django/forms/models.py

```diff
@@ -1343,6 +1414,62 @@ def inlineformset_factory(
     return FormSet
 
 
+class InlineFormSetMeta(ModelFormSetMeta):
+    """Meta class for creating inlineformset using Declarative Syntax."""
+
+    def __new__(cls, name, bases, attrs):
+        """Initialize the attributes given to the InlineFormSet class."""
```

Comment:
```suggestion
```
I would say some of these comments are not needed as the code is self explanatory